packages/manager/modules/dbaas-logs/src/logs/detail/index/logs-index.html

```diff
@@ -87,7 +87,7 @@ <h3 data-translate="logs_index_title"></h3>
                 href="{{ url.address }}"
                 data-external
                 ><span
-                    data-ng-bind="::'logs_index_access_' + url.type | translate"
+                    data-ng-bind="::'logs_index_access_os_api' | translate"
```

Comment:
prefer to use the translate dertive instead:
```suggestion
                    data-translate="logs_index_access_os_api"
```